backend/post-service/src/main/java/com/speakapp/postservice/controllers/CommentController.java

```diff
@@ -24,19 +23,20 @@ public class CommentController {
     private final CommentService commentService;
     private final JwtDecoder jwtDecoder;
     private static final String AUTH_HEADER_PREFIX = "Bearer ";
+
     @GetMapping("")
     public CommentPageGetDTO getCommentsForPostByCreatedAtSorted(
-            @RequestParam(defaultValue = "0") int pageNumber,
-            @RequestParam(defaultValue = "10") int pageSize,
+            @RequestParam(defaultValue = "0") int firstComment,
+            @RequestParam(defaultValue = "10") int lastComment,
             @RequestParam(defaultValue = "createdAt") String sortBy,
-            @RequestParam(defaultValue = "DESC") Sort.Direction sortDirection,
+            @RequestParam(defaultValue = "DESC") String sortDirection,
```

Comment:
A czemu tutaj String? bardziej enum by sie przydał

Reply:
dziś wieczorkiem to poprawię